packages/vite/src/node/plugins/css.ts

```diff
@@ -1771,6 +1814,7 @@ async function minifyCSS(
   css: string,
   config: ResolvedConfig,
   inlined: boolean,
+  concatCssEndLineMap?: Map<string, number>,
```

Comment:
```suggestion
  concatCssEndLines?: Array<{ file: string, end: number }>,
```
Would you change this to an array because the order has a meaning?

Reply:
Yeah I agree with you. I don't think it has to be a map since I am just iterating through.